cirq/google/engine/client/quantum_v1alpha1/gapic/enums.py

```diff
@@ -94,6 +94,23 @@ class Health(enum.IntEnum):
         DOWN = 2
         UNAVAILABLE = 3
 
+    class TimeSlotType(enum.IntEnum):
```

Comment:
In the API, the TimeSlotType nested inside of QuantumTimeSlot. Does this need to match to work appropriately with the gRPC client, or is the plan to translate these? I think that this code was originally generated from the API protos, so I'm guessing that they should be in sync.

Reply:
Done.